source/aws-ruby-sdk/error_helper.rb

```diff
@@ -0,0 +1,7 @@
+module IdentityIdpFunctions
+  class MisconfiguredLambdaError < StandardError
```

Comment:
can we namespace this to match the file?

ex `errors.rb` and it contains:

```
module IdentityIdpFunctions
  module Errors
    class MisconfiguredLambdaError < StandardError
```

Reply:
👍🏼